src/mesh.cpp

```diff
@@ -141,8 +141,8 @@ create_spoke_mesh(MPI_Comm comm, std::size_t target_dofs,
                     {0, 2, 3, 4}, {6, 7, 4, 2}, {2, 3, 4, 7}};
 
   // Calculate number of points and cells (only on process 0)
-  int npoints = 0;
-  int ncells = 0;
+  std::uint32_t npoints = 0;
```

Comment:
I know this is unsigned, but we've generally been using signed ints for consistency everywhere.